packages/configuration/src/ConfigurationService.ts

```diff
@@ -0,0 +1,42 @@
+import { ConfigurationService } from '@capsulajs/capsulajs-configuration-service/lib/api';
+import {
+  ConfigurationServiceHardcoreRemote,
+  ConfigurationServiceLocalStorage,
+  ConfigurationServiceFile,
+  ConfigurationServiceHttp,
+} from '@capsulajs/capsulajs-configuration-service/lib';
+// @ts-ignore
+import { ConfigurationConfig } from './api';
+
+// @ts-ignore
+export default (WORKSPACE: Workspace, SERVICE_COMNFIG: ConfigurationConfig) => {
```

Comment:
When we thought at the beginning that workspace and service config will be global variables we wrote them with capital letters, but now that they are just some arguments of the bootstrap function, we should follow the convention and write them `(workspace: Workspace, serviceConfig: ConfigurationConfig)`